using-timescaledb/compression.md

```diff
@@ -273,13 +296,21 @@ TimescaleDB does not support inserts or updates into a compressed chunk. To inse
 or update data, we must first decompress the target chunks. But before we can do
 that, we need to turn off our compression policy. Otherwise that policy will attempt
 to re-compress the chunks that we are currently working on (not the desired result).
-To accomplish this we will issue the following command:
+To accomplish this we first find the job_id of the policy using:
+
+```sql
+SELECT job_id, table_name 
+FROM _timescaledb_config.bgw_policy_compress_chunks p 
+INNER JOIN _timescaledb_catalog.hypertable h ON (h.id = p.hypertable_id);
+```
```

Comment:
To-do: feels like we could use a better UX command that doesn't require join to internal catalog

Reply:
agreed